sitback-furniture/src/components/cart-item/CartItem.module.scss

```diff
@@ -0,0 +1,71 @@
+.cartItem {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    padding: 1rem;
+    background-color: #fff;
+    border-radius: 15px;
+    width: 90%;
+    height: 80px;
+    margin: 30px;
+    position: relative;
+
+    img {
+        width: 125px;
+        height: 85px;
+        object-fit: cover;
+        border-radius: 12px;
+        position: absolute;
+        left: -30px;
+        top: 11px;
+    }
+
+    .itemContent {
+        margin-left: 100px;
+
+        h4 {
+            font-size: 14px;
+            font-weight: 400;
+        }
+
+        p {
+            font-size: 13px;
+        }
+    }
+
+    .itemActions {
+        position: absolute;
+        /* border: 1px solid black; */
+
+        height: 25px;
+        right: 20px;
+        bottom: -10px;
+        background-color: #fff;
+        box-shadow: 2px 2px 10px #c8c8c8;
+        border-radius: 3px;
+
+        .cartAction {
+            display: flex;
+            justify-content: space-between;
+            width: 60px;
+            margin: 3px 10px;
+            align-items: center;
+
+            span {
+                font-size: 12px;
+            }
+
+            .actions {
+                cursor: pointer;
+            }
+        }
+
+        button {
```

Comment:
could you let me know if we could get these styles in a configurable in Button component

Reply:
Moved the styles to button component